org.eclipse.jdt.ui.unittest.junit/.settings/.api_filters

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Unrelated change, please revert.

Reply:
I reverted the unwanted commit. Thank you!